TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/approve.py

```diff
@@ -0,0 +1,205 @@
+        status_result = service.get_workflow_status(
+            workflow_name=workflow_name,
+            namespace=namespace,
+            argo_server=argo_server,
+            token=token,
+            insecure=insecure
+        )
+
+        if not status_result['success']:
+            click.echo(f"Error getting workflow status: {status_result['error']}", err=True)
+            ctx.exit(ExitCode.FAILURE.value)
+
+        # Display workflow information
+        click.echo("\n" + "=" * 60)
+        click.echo(f"Workflow: {workflow_name}")
+        click.echo(f"Namespace: {namespace}")
+        click.echo(f"Phase: {status_result['phase']}")
+        if status_result['started_at']:
+            click.echo(f"Started: {status_result['started_at']}")
+        click.echo("=" * 60)
+
+        # Display steps and collect suspended steps
+        suspended_steps = []
+        if status_result['steps']:
+            click.echo("\nWorkflow Steps:")
+            click.echo("-" * 60)
+            for step in status_result['steps']:
+                step_type = step['type']
+                step_name = step['name']
+                step_phase = step['phase']
+
+                # Collect suspended steps for approval selection
+                if step_type == 'Suspend' and step_phase == 'Running':
+                    suspended_steps.append(step_name)
+                    click.echo(f"  > {step_name} [{step_type}] - {step_phase} (WAITING FOR APPROVAL)")
+                else:
+                    status_icon = "+" if step_phase == "Succeeded" else "*"
+                    click.echo(f"  {status_icon} {step_name} [{step_type}] - {step_phase}")
+            click.echo("-" * 60)
+
+        # Interactive confirmation unless --acknowledge is passed
+        if not acknowledge:
+            if not suspended_steps:
+                click.echo("\nNo suspended steps found waiting for approval.")
+                ctx.exit(ExitCode.SUCCESS.value)
+
+            click.echo("\nSelect an action:")
+            click.echo("-" * 60)
+
+            # Display suspended steps with numbers
+            for idx, step_name in enumerate(suspended_steps):
+                click.echo(f"  [{idx}] Approve and resume: {step_name}")
+
+            click.echo("  [c] Cancel")
+            click.echo("-" * 60)
+
+            # Get user selection
+            choice = click.prompt("\nEnter your choice", type=str).strip().lower()
+
+            if choice == 'c':
+                click.echo("Approval cancelled.")
+                ctx.exit(ExitCode.SUCCESS.value)
+
+            # Validate numeric choice
+            try:
+                choice_idx = int(choice)
+                if choice_idx < 0 or choice_idx >= len(suspended_steps):
+                    click.echo(f"Error: Invalid choice. Please select 0-{len(suspended_steps) - 1} or 'c'", err=True)
+                    ctx.exit(ExitCode.FAILURE.value)
+
+                selected_step = suspended_steps[choice_idx]
```

Comment:
are you missing filtering the resume down to just one node?  I expected to see resume called with an node identifier.